### ISSUE TITLE
Gun tweaks

### ISSUE THREAD

**Issue by TeslaCoil9001** [studio-minus/ppg-bugs-and-requests#2019]
**Feature description**
**I know I'm not supposed to multi-request but mestiez can just approve this and only add select features if he wants too.** 

-Slightly increase Submachine Gun and G1 Submachine Gun spread (Probably similar spread to Soviet Submachine Gun)
-Heavily increase Minigun spread (Probably close to LMG)
-Make LMG get hotter faster (It's waaaaay too slow)
-Make ejected shotgun shells red
-Make shotgun able to detach limbs instead of just gibing them
-Animate shotgun pump, revolver cylinder, grenade launcher cylinder, and fire extinguisher lever

**Feature purpose**
All of these lean towards realism and making the game more polished feeling.

**Comment by bibbybabby** [comment]
the shotgun looks more like a single barreled break action type than a pump action, and yellow shells do exist as 20 gauge. other than that this request is good

**Comment by TeslaCoil9001** [comment]
thx. it has a pump tho so it looks like a pump action

**Comment by bibbybabby** [comment]
> thx. it has a pump tho so it looks like a pump action

it's a grip but yea

**Comment by DindinYT37** [comment]
Still shouldn't multi-request but ok

**Comment by TeslaCoil9001** [comment]
> Still shouldn't multi-request but ok

Did you even read it

**Comment by TeslaCoil9001** [comment]
Yes I did. But I already knew not to multi request, but at the same time, I can't make each one an individual suggestion without getting ridiculed for spam. It's kind of a catch-22

**Comment by DindinYT37** [comment]
> Yes I did. But I already knew not to multi request, but at the same time, I can't make each one an individual suggestion without getting ridiculed for spam. It's kind of a catch-22

Well you wouldn't be ridiculed for spam if you took your time with each request, it's only spam if you just fill in the title, copy it into the issue and post it.

**Comment by TeslaCoil9001** [comment]
THANK YOU!!!
